### ISSUE TITLE
Searchable text in pdf

### ISSUE THREAD

**Issue by FilipAndersson245** [vercel/satori#198]
# Feature Request
It would be awsome if for pdf output texts stays at text to be easier to search etc.

## Description
For html output its simple to search the generated text, but it would be nice if for pdf's to also enabled you to mark and search text.

**Comment by shuding** [comment]
Yeah that PDF demo is not polished, many things are not working properly. I wonder if we can have a separate documentation about that, with some examples.

Searchable text requires embedding the font, which is not an easy thing.
